web-common/src/features/dashboards/filters/measure-filters/MeasureFilterMenu.svelte

```diff
@@ -3,23 +3,22 @@
   import InputV2 from "@rilldata/web-common/components/forms/InputV2.svelte";
   import Select from "@rilldata/web-common/components/forms/Select.svelte";
   import { getDimensionDisplayName } from "@rilldata/web-common/features/dashboards/filters/getDisplayName";
-  import { MeasureFilterOptions } from "@rilldata/web-common/features/dashboards/filters/measure-filters/measure-filter-options";
-  import { getStateManagers } from "@rilldata/web-common/features/dashboards/state-managers/state-managers";
   import {
-    createBetweenExpression,
-    createBinaryExpression,
-  } from "@rilldata/web-common/features/dashboards/stores/filter-utils";
+    MeasureFilterComparisonType,
+    MeasureFilterEntry,
+  } from "@rilldata/web-common/features/dashboards/filters/measure-filters/measure-filter-entry";
   import {
-    V1Operation,
-    type V1Expression,
-  } from "@rilldata/web-common/runtime-client";
+    MeasureFilterOperation,
+    MeasureFilterOptions,
+  } from "@rilldata/web-common/features/dashboards/filters/measure-filters/measure-filter-options";
+  import { getStateManagers } from "@rilldata/web-common/features/dashboards/state-managers/state-managers";
   import { createEventDispatcher } from "svelte";
   import { createForm } from "svelte-forms-lib";
   import * as yup from "yup";
 
   export let dimensionName: string;
   export let name: string;
-  export let expr: V1Expression | undefined;
+  export let filter: MeasureFilterEntry | undefined;
```

Comment:
Same thing. Can we get rid of the undefined and just wrap the use of this component in an `#if` block?